libs/langchain/langchain/chat_loaders/imessage.py

```diff
@@ -95,6 +95,8 @@ def lazy_load(self) -> Iterator[chat_loaders.ChatSession]:
         """
 
         try:
+            import sqlite3
```

Comment:
is line 40 also supposed to have an import

Reply:
also think you've got to import outside the try/except if you're catching a sqlite3 error

Reply:
Haha my lint formatter is over-zealous. Thanks for flagging